ci/maketmpl/util.go

```diff
@@ -34,3 +35,25 @@ func addFileToZip(z *zip.Writer, file, pathInZip string) error {
 	}
 	return nil
 }
+
+func buildZip(dir string, files []string, out string) error {
```

Comment:
Thoughts for future: might have been slightly nicer to use `os.DirFS` & `zipfile.Writer.AddFS()` for this?

Reply:
well, I actually need to ignore some files.. so I will need to write custom fs's anyways.. feel that it is more flexible, direct and straightforward this way.